Source/misdat.cpp

```diff
@@ -383,7 +383,7 @@ void LoadMisdat()
 	}
 
 	// Sanity check because we do not actually parse the IDs yet.
-	assert(static_cast<size_t>(MissileID::LAST) + 1 == MissilesData.size());
+	assert(95 == MissilesData.size() || static_cast<size_t>(MissileID::LAST) + 1 == MissilesData.size());

```

Comment:
Maybe we should add `MissileID::LastDiablo`, like we do for `SpellID`?

https://github.com/diasurgical/DevilutionX/blob/f1c8aa22f11404411f4b0acb77248039118da6ba/Source/spelldat.h#L72

Reply:
Probably for the better considering it uncovered a mistake I made